contrib/database/sql/conn_test.go

```diff
@@ -0,0 +1,95 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package sql
+
+import (
+	"context"
+	"database/sql/driver"
+	"log"
+	"testing"
+
+	"github.com/go-sql-driver/mysql"
+	"github.com/lib/pq"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
```

Comment:
You've missed this change in my diff: this import needs to be in its own group, under `testing` and have an empty line above and below it.

We list imports in 3 groups, separated by empty lines:
1. Standard library
2. Current project (dd-trace-go)
3. External dependencies